src/core/env/EnvContext.tsx

```diff
@@ -1,13 +1,12 @@
-import { createContext, FC, ReactNode } from 'react';
+import { createContext, FC, PropsWithChildren } from 'react';
 
 import Environment from './Environment';
 
 const EnvContext = createContext<Environment | null>(null);
 
-interface EnvProviderProps {
-  children: ReactNode;
+type EnvProviderProps = PropsWithChildren & {
   env: Environment;
-}
+};
```

Comment:
Is this something that is recommended to use instead of writing out `children: ReactNode` in a type? Have never seen before, it looks convenient!

Reply:
It basically means `children: ReactNode | undefined` and I guess the `undefined` is needed here, but I guess I could have also written it out.